winlogtimeline/collector/filter.py

```diff
@@ -8,15 +8,35 @@ def filter_logs(project, config):
     :param config: A config dictionary.
     :return: A list of logs that satisfy the filters specified in the configuration.
     """
+    #for r in project._conn.execute('PRAGMA table_info(logs);'):
+    #    print(r)
+
+    headers = Record.get_headers()
+    col, operator, value = config[0]
+    if operator == 'Contains':
+        operator = 'like'
+        value = "'%" + value + "%'"
+
+
+    info = project._conn.execute('PRAGMA table_info(logs);') #(id, column name, type, not null, pk, auto-inc)
+    id, columns, types, n, p, a = zip(*info)
```

Comment:
Try to avoid using `id` and `type` as variable names, as they're both builtin names. It shouldn't cause any issues here, but I'd like to see that changed in future commits.